README.md

```diff
@@ -2,84 +2,83 @@
 # Amazon EKS SSP Quick Start
 
 ![GitHub](https://img.shields.io/github/license/aws-quickstart/quickstart-ssp-amazon-eks)
-![Build](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiTWxBQzVUcTBvdSsvbE9mR0ZWeTJjbi96OUpBREorSG51UjMzQ1UyNXdmUzZ2dUJoTkhIODFJWjN2QjRGcnhWS0pYLzFQRU5uOThiUEp1WjEzS0htbUpVPSIsIml2UGFyYW1ldGVyU3BlYyI6IlRkUFRoTWtjdElBMkR5NEMiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
+![Build](https://codebuild.us-west-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiYVN6RlEyQmNNRVRxRWYxUkhBcStQcHFkTExZa2dtZXBWcDdqZ2lWNCtXVENOcWwzV0F0cklUOXlnVGtMZ1BzR0JLZUtHU2V3dUcwb0NOMmdxWGxKOFpVPSIsIml2UGFyYW1ldGVyU3BlYyI6IlEyWmdVeXlxMS9UOVk0QUMiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
 
 Welcome to the `Amazon EKS SSP Quickstart` repository.
 
-This repository contains the source code for the [`ssp-amazon-eks`](https://www.npmjs.com/package/@aws-quickstart/ssp-amazon-eks) NPM module. `ssp-amazon-eks` is a [CDK](https://aws.amazon.com/cdk/) construct that makes it easy for customers to build and deploy a Shared Services Platform (SSP) on top of [Amazon EKS](https://aws.amazon.com/eks/).
+This repository contains source code for the [`cdk-eks-blueprint`](https://www.npmjs.com/package/@shapirov/cdk-eks-blueprint) NPM module. The `cdk-eks-blueprint` module is a [CDK](https://aws.amazon.com/cdk/) construct for customers to build and deploy a Shared Services Platform (SSP) on top of [Amazon EKS](https://aws.amazon.com/eks/).
```

Comment:
Us the new @aws-quickstart module we have and the new lib name `ssp-amazon-eks`.

Reply:
Fixed.